src/iden3comm/handlers/credential-proposal.ts

```diff
@@ -165,6 +165,7 @@ export class CredentialProposalHandler
   constructor(
     private readonly _packerMgr: IPackageManager,
     private readonly _identityWallet: IIdentityWallet,
+    private readonly _credWallet: ICredentialWallet,
```

Comment:
looks like breaking change, can't we make getter for credential wallet in identity wallet instance? or create function in identity wallet that will proxy call for query?